howso/utilities/feature_attributes/relational.py

```diff
@@ -622,117 +634,143 @@ def _infer_datetime_attributes(self, feature_name: str) -> Dict:
             dt_format = ISO_8601_FORMAT
 
         return {
-            'type': 'continuous',
+            'type': feature_type_override if feature_type_override else 'continuous',
             'data_type': 'formatted_date_time',
             'date_time_format': dt_format,
         }
 
-    def _infer_date_attributes(self, feature_name: str) -> Dict:
+    def _infer_date_attributes(
+        self,
+        feature_name: str,
+        feature_type_override: Optional[str] = None
+    ) -> Dict:
         # Although rare, it is plausible that a date field could be a
         # primary- or foreign-key.
         if (
                 self._is_primary_key(feature_name) or
                 self._is_foreign_key(feature_name)
-        ):
+        ) or feature_type_override == 'nominal':
             return {
                 'type': 'nominal',
             }
+        else:
+            return {
+                'type': feature_type_override if feature_type_override else 'continuous',
+                'data_type': 'formatted_date_time',
+                'date_time_format': ISO_8601_DATE_FORMAT,
+            }
 
-        return {
-            'type': 'continuous',
-            'data_type': 'formatted_date_time',
-            'date_time_format': ISO_8601_DATE_FORMAT,
-        }
+    def _infer_time_attributes(
+        self,
+        feature_name: str,
+        feature_type_override: Optional[str] = None
+    ) -> Dict:
 
-    def _infer_time_attributes(self, feature_name: str) -> Dict:
         return {
-            'type': 'continuous',
-            'data_type': 'number',
+            'type': feature_type_override if feature_type_override else 'continuous',
+            'data_type': 'number'
         }
 
-    def _infer_timedelta_attributes(self, feature_name: str) -> Dict:
+    def _infer_timedelta_attributes(
+        self,
+        feature_name: str,
+        feature_type_override: Optional[str] = None
+    ) -> Dict:
         # Although rare, it is plausible that a timedelta field could be a
         # primary- or foreign-key.
         if (
                 self._is_primary_key(feature_name) or
                 self._is_foreign_key(feature_name)
-        ):
+        ) or feature_type_override == 'nominal':
             return {
                 'type': 'nominal',
             }
 
         return {
-            'type': 'continuous',
-            'data_type': 'number',
+            'type': feature_type_override if feature_type_override else 'continuous',
+            'data_type': 'number'
         }
 
-    def _infer_boolean_attributes(self, feature_name: str) -> Dict:
+    def _infer_boolean_attributes(
+        self,
+        feature_name: str,
+        feature_type_override: Optional[str] = None
+    ) -> Dict:
+        if feature_type_override in ('continuous', 'ordinal'):
```

Comment:
Might be safer to check feature_type_override is != nominal in case new types are added later.